script.test.js

```diff
@@ -0,0 +1,55 @@
+import {
+  checkNormOrMorse,
+  invalidCharCheck,
+  convertNormToMorse,
+  convertMorseToNorm,
+} from "./functionScripts.js";
+
+describe("Tests whether the input from the user is Normal Text, Morse Code or an invalid character", () => {
+  it("Should return 'Morse Code', if input text starts with either '.' or '-'", () => {
+    expect(checkNormOrMorse(".--.--")).toBe("Morse Code");
+    expect(checkNormOrMorse(".00029")).toBe("Morse Code");
+    expect(checkNormOrMorse("-12")).toBe("Morse Code");
+  });
+
+  it("Should return 'Normal Text', if input text is every other character'", () => {
+    expect(checkNormOrMorse("hello world")).toBe("Normal Text");
+    expect(checkNormOrMorse("0900-/.")).toBe("Normal Text");
+  });
+
+  it("Should throw an error for invalid characters", () => {
+    expect(checkNormOrMorse("*")).toBe("Error");
+    expect(checkNormOrMorse("~")).toBe("Error");
+    expect(checkNormOrMorse("}")).toBe("Error");
+  });
```

Comment:
A test here for space character would have quickly figured out if normal. Also change name to 'return error' or throw the error and test for a thrown error in function.